docs/reference/api-gateway/ui.md

```diff
@@ -0,0 +1,34 @@
+# API Gateway via the UI
+
+#### In this document
+
+- [No Authentication](#none-auth)
+- [Basic Authentication](#basic-auth)
+
+Creating API Gateways from the UI is really simple.
+Inside your project screen, go to the *"API Gateways"* tab, and click *"NEW API Gateway"*.
```

Comment:
Inside your project page, go to the **API Gateways** tab, and click **NEW API Gateway**.